internal/web_server/web/views/pages/dashboard_pages/labels.templ

```diff
@@ -0,0 +1,44 @@
+package dashboard_pages
+
+import (
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/models/labels"
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/dashboard_components/label_components"
+)
+
+templ LabelsPage(labels []labels.Label) {
+	@layout_components.DashBoardLayout("Etiketter") {
+		<div class="flex flex-col gap-6 px-8 py-6 max-w-screen-md mx-auto">
+			<section>
+				<h1
+					class="text-3xl font-bold text-gray-800 mb-2"
+				>Ettiketter</h1>
```

Comment:
You call it "kategorier" on EditQuiz page, but here you call it "etiketter". Be consistent.

Reply:
Also, typo: `ettiketter` -> `etiketter`